Plots/Vectors/vector_1.py

```diff
@@ -0,0 +1,149 @@
+"""
+vector_1
+========
+
+Plot U & V vector over SST
+
+https://www.ncl.ucar.edu/Applications/Scripts/vector_1.ncl
+"""
+
+###############################################################################
+# Import necessary packages
+import xarray as xr
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+import cartopy
+import cartopy.crs as ccrs
+import cmaps
+
+###############################################################################
+# Read in data from netCDF files
+sst_in = xr.open_dataset('../../data/netcdf_files/sst8292.nc')
+uv_in = xr.open_dataset('../../data/netcdf_files/uvt.nc')
+
+###############################################################################
+# Extract required variables from files
+
+# Indices of January 1988 in each file
+date_sst = sst_in['date']
+date_uv = uv_in['date']
+ind_sst = np.where(date_sst == 198801)[0][0]
+ind_uv = np.where(date_uv == 198801)[0][0]
+
+# Read in grid information
+lat_sst = sst_in['lat']
+lon_sst = sst_in['lon']
+lat_uv = uv_in['lat']
+lon_uv = uv_in['lon']
+
+# Read SST and U, V
+sst = sst_in['SST'].isel(time=ind_sst)
+u = uv_in['U'].isel(time=ind_uv, lev=0)
+v = uv_in['V'].isel(time=ind_uv, lev=0)
```

Comment:
Can we use `sel` here, instead of `isel`?  I realize that this matches explicitly what is done in the NCL script, but that's just because NCL doesn't have this functionality.

Reply:
I've made this change -- it reduces the python line count (and avoids variables like `ind_sst`), though it adds a little complication in that we need `date` to be a dimension. Honestly, that's an issue with the formatting of the original `nc` file rather than anything code-related...